tools/windows/resource.rc.in

```diff
@@ -0,0 +1,32 @@
+#include "winresrc.h"
+
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION @VERSION_MAJOR@,@VERSION_MINOR@,@VERSION_PATCH@
+ PRODUCTVERSION @VERSION_MAJOR@,@VERSION_MINOR@,@VERSION_PATCH@
+ FILEFLAGSMASK VS_FFI_FILEFLAGSMASK
+#ifdef DEBUG
+ FILEFLAGS 1
+#else
+ FILEFLAGS 0
+#endif
+ FILEOS VOS_NT
+ FILETYPE VFT_DLL
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "CompanyName", "\0"
+            VALUE "FileDescription", "@TARGET_NAME@ @VERSION_STRING@\0"
+            VALUE "InternalName", "@TARGET_NAME@-@VERSION_MAJOR@.dll\0"
+            VALUE "LegalCopyright", "Copyright (C) 2007-2018 Laurent Gomila\0"
+            VALUE "OriginalFilename", "@TARGET_NAME@-@VERSION_MAJOR@.dll\0"
+            VALUE "ProductName", "Simple Fast Multimedia Library Module\0"
```

Comment:
Small idea: maybe `"Simple Fast Multimedia Library @TARGET_NAME Module\0"` ?

Reply:
@Foaly well, that would translate to 'Simple Fast Multimedia sfml-audio Module' for audio, for example. I could prolly switch the sfml_add_library arg to accept 'Audio' and so forth and do
some magic to get 'sfml-audio' out of it where appropriate.

Reply:
Ah I was thinking it contained "Audio" directly. Not sure if its worth the effort then, since all the other values contain the submodule name.